refresh_modules.py

```diff
@@ -0,0 +1,276 @@
+import json
```

Comment:
I think we should keep this generator script out-of-tree since this is not going to be used by the end-user.

Reply:
At this stage, it's still experimental. But Yes, this is something we've to speak about. Technically speaking, I don't think we've a strong relationship with VMware, but I've yet to compare with some other Swagger outputs. If it's really the case, we can prepare a new project.